packets/packets.go

```diff
@@ -579,7 +580,15 @@ func parseTLV(data []byte) (result []interface{}, err error) {
 		case tlvTIMESTAMP: // timestamps without units
 			x := binary.BigEndian.Uint16(data[2:])
 			y := binary.BigEndian.Uint32(data[4:])
-			result = append(result, MakeTimestamp(x, y, 0.0))
+
+			// we assume a 64 bit int with units of nanoseconds
```

Comment:
I guess I can approve this, but I'm a little concerned that we'll be making life more difficult in the future. The hope is that no timestamp source that has any meaningful units would use the `tlvTIMESTAMP` type but would always favor `tlvTIMESTAMPUNIT`. 

What was wrong with the previous choice to set `PacketTimestamp.Rate=0`? Did something break when we did that?

Reply:
(I could totally imagine that Rate=0 breaks something.)